include/locales/en-US.h

```diff
@@ -12,6 +12,12 @@
 #define LANG_CANCEL "Cancel"
 #define LANG_SAVE "Save"
 
+// App: Timer
+#define LANG_TIMER "Timer"
+#define LANG_START "Start"
+#define LANG_PAUSE "Pause"
+#define LANG_RESUME "Resume"
+
```

Comment:
We need to devise a way for an application to fall back to the English language if it has not been localized yet.

Reply:
I think your idea is good, but it should be opened as a separate issue and implemented in a separate branch.
This PR only implements the Timer feature.

Reply:
Fixed the error